extensions/notebook/src/common/utils.ts

```diff
@@ -16,7 +19,7 @@ export function getLivyUrl(serverName: string, port: string): string {
 }
 
 export async function mkDir(dirPath: string, outputChannel?: vscode.OutputChannel): Promise<void> {
-	if (!await fs.exists(dirPath)) {
+	if (!await fs.pathExists(dirPath)) {
```

Comment:
This is the only functional change. The typings for fs-exta says exists isn't a method, instead pathExists seems to be the equivalent. Any advice on how to test this would be great.

Reply:
Create a new notebook using crtl+shift+N -> change any *spark* kernel.

Reply:
Alright everything seems to work fine.